pkg/agent/variables.go

```diff
@@ -111,7 +111,7 @@ func getOutBoundCmd(cs *api.ContainerService) string {
 	if cs.GetCloudSpecConfig().CloudName == api.AzureChinaCloud {
 		registry = `gcr.azk8s.cn 443`
 	} else if cs.IsAKSCustomCloud() {
-		registry = cs.Properties.CustomCloudEnv.McrURL
+		registry = cs.Properties.CustomCloudEnv.McrURL + " 443"
```

Comment:
originally I thought this 443 should go to McrURL as well that is why I didn't put the port here, One a second thought, this is the correct way of implementation as we may need to use McrURL in different places without port.